sycl/tools/sycl-prof/main.cpp

```diff
@@ -46,6 +46,7 @@ int main(int argc, char **argv, char *env[]) {
   NewEnv.push_back("XPTI_SUBSCRIBERS=libsycl_profiler_collector.so");
   NewEnv.push_back("XPTI_TRACE_ENABLE=1");
   NewEnv.push_back("ZE_ENABLE_TRACING_LAYER=1");
+  NewEnv.push_back("SYCL_PI_CUDA_ENABLE_TRACING=1");
```

Comment:
What effect does this environment variable have? I don't see it used anywhere.

Reply:
Thank you for noticing it, I missed that.

I think that originally this environment variable was meant to disable the tracing capabilities at runtime , i.e., the calling of `{enable,disable}CUDATracing()` in `sycl/plugins/cuda/pi_cuda.cpp`.

A similar variable for level zero (`SYCL_PI_LEVEL_ZERO_ENABLE_TRACING`) was removed in https://github.com/intel/llvm/pull/6023. I did the same with `SYCL_PI_CUDA_ENABLE_TRACING`. Basically, these variables are replaced by checking `xptiTraceEnabled()`.